### DISCUSSION TITLE
Qlipper looks not maintained

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2372] [General]
https://github.com/pvanek/qlipper/pull/143
It more than a year since the last merge. Didn't have someone write access (@palinek )?

**Comment by palinek** [comment]
Yes, I have write access. Merged. But yes, qlipper is more-or-less unmaintained :-(

**Reply by stefonarch** [reply]
Would it be possible to transfer it  here?

**Reply by palinek** [reply]
I don't know if moving repo in github is possible and what are the requirements, if even it is. But definitely we would need the owner (@pvanek) for it.

**Reply by stefonarch** [reply]
Afaik it would be easy:

>  Transfer ownership
>
>Transfer this repository to another user or to an organization where you have the ability to create repositories.


>But definitely we would need the owner (@pvanek) for it.

 Sure.

**Reply by pvanek** [reply]
hi, I'm ready to transfer the ownership. Where to?

**Reply by stefonarch** [reply]
For me it would fit fine under LXQt's hood.  @palinek @tsujan @yan12125 @luis-pereira ?

**Reply by yan12125** [reply]
Sure, that's good.

**Reply by stefonarch** [reply]
This is still open. Any objections to take it under our hood?

**Reply by palinek** [reply]
I've created a fresh new repo https://github.com/lxqt/lxqt-clip with rebranded Qlipper. Can you have a look, if we can go forward with it?

**Reply by stefonarch** [reply]
Didn't find any issue, go ahead. 
I just noticed I'm now too much tied to copyq  because
1. having my screenshots pasted automatically in the clipboard 
2. search items

but that's another story.

**Reply by stefonarch** [reply]
I think we should replace qlipper with lxqt-clip on weblate then.

**Reply by palinek** [reply]
1. You can use similar
```sh
#!/usr/bin/bash
shot_file="$HOME/tmp/downloads/screenshot.png"
if [[ -n "$WAYLAND_DISPLAY" ]]; then
    [[ ${#@} -ne 0 ]] || g_args=(-g "$(slurp -d)")
    grim -t png "${g_args[@]}" "$shot_file" && wl-copy -t image/png < "$shot_file" && xdg-open "$shot_file"
else
    import "$@" "$shot_file" && xclip -selection clipboard -t image/png -i "$shot_file" && xdg-open "$shot_file"
fi
```
```xml
    <keybind key="A-Print">
      <action name="Execute" command="screenshot.sh" />
    </keybind>
    <keybind key="Print">
      <action name="Execute" command="screenshot.sh -window, root" />
    </keybind>
```
2. no help with qlipper/lxqt-clip

> Didn't find any issue, go ahead.

What does _go ahead_ mean here? IMO, it is ready to use/ship/release.

**Reply by stefonarch** [reply]
"go ahead" is the aswer to "can we go forward", didn't know what exactly you meant.

But why wasn't https://github.com/pvanek/qlipper/pull/147/changes merged before? Maybe this could be done/integrated before releasing.

To 1.  thanks, but I'm using the integrated screenshot tool (region/window/fullscreen) in niri which saves to /tmp and to clipboard automatically.

**Reply by tsujan** [reply]
> I've created a fresh new repo https://github.com/lxqt/lxqt-clip with rebranded Qlipper.

This is very good news. CopyQ is too advanced for many users.

So, we'll have a new package in our releases?

**Reply by palinek** [reply]
> So, we'll have a new package in our releases?

If you bless it and make the release :-)

**Reply by stefonarch** [reply]
Sure, but let translators update some translations before.

**Comment by dglent** [comment]
I use [copyq](https://github.com/hluk/CopyQ) with LXQt
I prefer this clipboard manager, i find comfortable the automatic paste of pinned entries
It is very full-featured but personally i have a very basic usage...

**Comment by 0323pin** [comment]
I was trying to be ahead of the curve and get this packaged for NetBSD before it is actually released.

But `CMakeLists.txt`  has `option(USE_SYSTEM_QXT "Use system Qxt Library for global shortcuts; off for Qt6" OFF)` hence, I get

```
-- Could NOT find Qt6LinguistTools (missing: Qt6LinguistTools_DIR)
CMake Error at CMakeLists.txt:30 (find_package):
  Found package configuration file:

    /usr/pkgsrc/wip/lxqt-clip/work/.buildlink/qt6/lib/cmake/Qt6/Qt6Config.cmake

  but it set Qt6_FOUND to FALSE so package "Qt6" is considered to be NOT
  FOUND.  Reason given by package:

  Failed to find required Qt component "LinguistTools".

  Expected Config file at
  "/usr/pkgsrc/wip/lxqt-clip/work/.buildlink/qt6/lib/cmake/Qt6LinguistTools/Qt6LinguistToolsConfig.cmake"
  does NOT exist
```

**Reply by Chiitoo** [reply]
Works for me here (on Gentoo Linux).

Is LinguistTools actually available?

I don't see how the `Qxt` option would be related there.

**Reply by 0323pin** [reply]
Thanks for reporting a successful test.

> Is LinguistTools actually available?

AFAIK, it's part of the `qttools` package. But yes, it is ...
```
~> ls /usr/pkg/lib/pkgconfig/ | grep Qt6Ling
lrwxr-xr-x root wheel  41 B  Thu Apr 30 18:37:41 2026 Qt6Linguist.pc ⇒ /usr/pkg/qt6/lib/pkgconfig/Qt6Linguist.pc
```

Wounder why I'm getting.
```
  Found package configuration file:

    /usr/pkgsrc/wip/lxqt-clip/work/.buildlink/qt6/lib/cmake/Qt6/Qt6Config.cmake

  but it set Qt6_FOUND to FALSE so package "Qt6" is considered to be NOT
  FOUND.
```

**Reply by 0323pin** [reply]
Ok, I found the reason.

I had `qttools` as a make dependency. If, I switch that to a runtime dependency, the build works.

**Reply by Chiitoo** [reply]
I see!

Indeed, part of qttools, though it /can/ be built without the linguist part but I imagine most packages have it enabled (Gentoo has a USE-flag for disabling it, for example).

**Reply by 0323pin** [reply]
@Chiitoo Thanks! You made me think one step further and everything is working now.
